src/NosCore.Algorithm/MpService/MpService.cs

```diff
@@ -11,7 +11,7 @@ namespace NosCore.Algorithm.MpService
 {
     public class MpService : IMpService
     {
-        private readonly long[,] _mpData = new long[Constants.ClassCount, Constants.MaxLevel];
+        private readonly long[,] _mpData = new long[Enum.GetNames(typeof(CharacterClassType)).Length, Constants.MaxLevel];
```

Comment:
just hardcode 5 if we change to 6 the amount of class we will still need to implement it so there is no real reason to get the number from the enum